src/hydra/fetchResource.ts

```diff
@@ -7,7 +7,7 @@ export default (
 ): Promise<any> => {
   return fetchJsonLd(
     resourceUrl,
-    Object.assign({ itemsPerPage: 0 }, options)
+    { itemsPerPage: 0 , ...options}
```

Comment:
Actually, passing `itemsPerPage` looks like an (existing) bug. It should be added to the URL's query parameters (user the `URL` class), not passed as an option to fetch. It's why tests fail.

Reply:
Seemlingy simply removing it here is fine then ? I've update the PR.
Thanks.